plugins/package-curation-providers/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+plugins {
+    // Apply core plugins.
+    `java-platform`
+    `maven-publish`
+}
+
+javaPlatform {
+    allowDependencies()
+}
+
+dependencies {
+    api(project(":plugins:package-curation-providers:clearly-defined"))
+    api(project(":plugins:package-curation-providers:file"))
+    api(project(":plugins:package-curation-providers:ort-config"))
+    api(project(":plugins:package-curation-providers:sw360"))
+}
+
+configure<PublishingExtension> {
+    publications {
+        create<MavenPublication>(name) {
+            groupId = "org.ossreviewtoolkit"
```

Comment:
Do we need the same `getGroupId()` logic here? Or should we use it anyway for consistency?

Reply:
I have hard-coded the correct value for now.